source/adapters/hip/context.hpp

```diff
@@ -33,22 +33,22 @@ typedef void (*ur_context_extended_deleter_t)(void *UserData);
 /// with a given device and control access to said device from the user side.
 /// UR API context are objects that are passed to functions, and not bound
 /// to threads.
-/// The ur_context_handle_t_ object doesn't implement this behavior. It only
-/// holds the HIP context data. The RAII object \ref ScopedContext implements
-/// the active context behavior.
 ///
-/// <b> Primary vs UserDefined context </b>
+/// Since the ur_context_handle_t can contain multiple devices, and a `hipCtx_t`
```

Comment:
I think I did this with `hipCtx_t` because I seem to remember some docs being generated from this (see the `<b><\b>` as well). And I think the underscore doesn't work without that (maybe)

Reply:
Thanks @keryell !